kahuna/public/js/upload/dnd-uploader.js

```diff
@@ -170,9 +179,22 @@ dndUploader.directive('dndUploader', ['$window', 'delay', 'safeApply', 'track',
                         ctrl.importing = false;
                     });
                     track.action(trackEvent, dropAction('Witness'));
-                } else {
+                } else if (uri) {
+                    ctrl.importing = true;
+                    ctrl.loadUriImage(uri)
+                        .catch(error => {
+                            if (error.body && error.body.errorMessage) {
+                                $window.alert(error.body.errorMessage);
```

Comment:
@tsop14 [for reference](https://github.com/guardian/grid/blob/2ef8bf576dd4aad8849d8d3e8cdbf5477664a960/kahuna/public/js/upload/jobs/upload-jobs.js#L98-L100). We could probably have these in a value store to ensure consistency.